Presentation.Web/app/services/Organization/organization-unit-service.ts

```diff
@@ -1,13 +1,26 @@
 ﻿module Kitos.Services.Organization {
 
-    export interface IOrganizationRegistrationsService {
-        getRegistrations(orgId: string, unitId: string): ng.IPromise<Models.Api.Organization.OrganizationUnitRegistrationDetailsDto>;
-        deleteSelectedRegistrations(orgId: string, unitId: string, body: Models.Api.Organization.OrganizationUnitRegistrationChangeRequestDto): angular.IPromise<boolean>;
-        deleteOrganizationUnit(organizationId: string, unitId: string): angular.IPromise<boolean>;
-        transferSelectedRegistrations(orgId: string, unitId: string, body: Models.Api.Organization.TransferOrganizationUnitRegistrationRequestDto): angular.IPromise<void>;
+    export interface IOrganizationUnitService {
+        getUnitAccessRights(orgUuid: string, unitUuid: string): ng.IPromise<Models.Api.Organization.UnitAccessRightsDto>;
+        getRegistrations(orgUuid: string, unitUuid: string): ng.IPromise<Models.Api.Organization.OrganizationUnitRegistrationDetailsDto>;
+        deleteSelectedRegistrations(orgUuid: string, unitUuid: string, body: Models.Api.Organization.OrganizationUnitRegistrationChangeRequestDto): angular.IPromise<boolean>;
+        deleteOrganizationUnit(organizationId: string, unitUuid: string): angular.IPromise<boolean>;
+        transferSelectedRegistrations(orgUuid: string, unitUuid: string, body: Models.Api.Organization.TransferOrganizationUnitRegistrationRequestDto): angular.IPromise<void>;
     }
 
-    export class OrganizationRegistrationsService implements IOrganizationRegistrationsService {
+    export class OrganizationUnitService implements IOrganizationUnitService {
+        getUnitAccessRights(orgUuid: string, unitUuid: string): ng.IPromise<Models.Api.Organization.UnitAccessRightsDto> {
+            return this
+                .$http
+                .get<API.Models.IApiWrapper<any>>(`api/v1/organizations/${orgUuid}/organization-units/${unitUuid}/access-rights`)
+                .then(
+                    result => {
+                        var response = result.data as { response: Models.Api.Organization.UnitAccessRightsDto }
+                        return response.response;
```

Comment:
use the ApiWrapper in stead of $http. That will ensure that you don't duplicate it's logic here

![image](https://user-images.githubusercontent.com/13116889/201864168-8ce440f6-17e4-4185-a3c8-a9a503a14e97.png)

Example of usage:
![image](https://user-images.githubusercontent.com/13116889/201864259-ed008079-5725-42cf-beca-fba8eb9c3fa3.png)